app/models/flexible_criterion.rb

```diff
@@ -100,6 +100,16 @@ def get_weight
     1
   end
 
+  # Returns the value entered when the criterion is set up.
+  def mark_set_up_value
+    max_mark
+  end
+
+  # Returns the maximum mark for a particular criterion.
+  def max_mark
```

Comment:
One name may be better than the other depending on the context, so I think it could improve code readability. I was thinking that the 'mark_set_up_value' method may be used in contexts where we either get the weight of a rubric criterion or the max of a flexible criterion. While, the 'max_mark' method may be used when we want to get the maximum mark that could be achieved for either a flexible or a rubric criterion.